### ISSUE TITLE
"cURL: no route to host"

### ISSUE THREAD

**Issue by kevinquillen** [lando/lando#1196]
I am testing a switch from DrupalVM to Lando for a Drupal 7 site. 

Everything has gone well so far (after some initial drush/database connection confusion) but I am running into an error that I don't get on my DrupalVM environment, but do with Lando.

We have part of the site that uses Guzzle to fetch data from a web service. On Lando, I get a cURL error:

`cURL error 7: Failed to connect to webservice.internalsite.com port 80: No route to host`

I _think_ this is because it cannot see our internal network (must be connected to our network to access/see them) but I don't know how to get this going with Lando or Docker. How can I resolve this?

**Comment by kevinquillen** [comment]
Hmm.. I tried this at home (outside of the office network) and it worked. I tried both with the proxy off and with it on - both cases work. So perhaps this is not a lando issue? Is there a way to diagnose why it does not work on our office network?

**Comment by kevinquillen** [comment]
I was able to fix this by editing my daemon.json for docker with the following values:

```
{
  "fixed-cidr-v6" : "2001:db8::/64",
  "fixed-cidr" : "192.168.1.5/25",
  "default-gateway-v6" : "2001:db8:abcd::89",
  "dns" : [
    "8.8.8.8"
  ],
  "bip" : "192.168.1.5/24"
}
```

I destroyed and started the lando project over, and now curl works.
